src/components/Formik/FormikCheckboxInput/FormikCheckboxInput.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {
+  FC,
+  ChangeEvent,
+  FocusEvent,
+} from 'react';
+import CheckboxInput from '../../CheckboxInput/CheckboxInput';
+
+interface FormikCheckboxInputProps {
+  field: {
+    id: string;
+    label: string;
+    name: string;
+    onBlur: (event: FocusEvent<HTMLElement>) => void;
+    onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+    value?: boolean;
+  };
+  form: { [key: string]: any; }; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Hey @JCarmine I could be wrong but I actually think that all of these params have type signatures that we can extend here. https://github.com/formium/formik/blob/master/packages/formik/src/types.tsx